context/context.go

```diff
@@ -287,7 +288,12 @@ func (context *AptlyContext) _database() (database.Storage, error) {
 	if context.database == nil {
 		var err error
 
-		context.database, err = goleveldb.NewDB(context.dbPath())
+		if context.config().DatabaseEtcd != "" {
```

Comment:
I wonder if we should have a more explicit way to pick the database, rather than look at `DatabaseEtcd`.

Maybe having a new config like `databaseBackend` to be one of: "leveldb", "etcd", etc.? What do you think?